proxy/src/project.rs

```diff
@@ -85,5 +90,14 @@ pub fn get(peer: &coco::PeerApi, project_urn: &coco::Urn) -> Result<Project, err
     let project = coco::get_project(peer, project_urn)?;
     let stats = coco::with_browser(peer, project_urn, |browser| Ok(browser.get_stats()?))?;
 
-    Ok((project, stats).into())
+    let maintainers: Vec<coco::User> = project
+        .maintainers()
+        .iter()
+        .map(|user_urn| {
+            let draft = coco::get_user(peer, user_urn);
+            draft.and_then(coco::verify_user)
+        })
+        .collect::<Result<_, _>>()?;
```

Comment:
Turbofish is banned right? What is a prettier way to do this?

Reply:
I use it all the time. Pretty sure it's not banned.

Reply:
Turbofish is great :fish: